api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -455,6 +456,20 @@ message Admin {
   // Indicates whether :ref:`global_downstream_max_connections <config_overload_manager_limiting_connections>`
   // should apply to the admin interface or not.
   bool ignore_global_conn_limit = 6;
+
+  // List of admin paths that are accessible. If not specified, all admin endpoints are accessible.
+  // When specified, only paths in this list will be accessible, all others will return 403 Forbidden.
```

Comment:
minor nit - if you want this on a separate line (i think it probably should) it needs a space above - otherwise rst will make it all one line